app/src/components/landing/ruleTemplates/RulePreviewModal/index.js

```diff
@@ -35,15 +35,15 @@ const RulePreviewModal = ({ rule, isOpen, toggle }) => {
         return;
       }
     }
-    const lastModifiedBy = Date.now();
+    const lastModified = Date.now();
     const modificationDate = Date.now();
```

Comment:
`lastModified` and `modificationDate` are same thing. Can you please check why are we using them?

Reply:
IIRC `modificationDate` was always part of the rule spec, but `lastModified` was added for entities that were introduced to workspaces. Keeping both here seems necessary. (to support both legacy code + new functionalities around team workspaces)
Although doesn't look relevant in the context of rule templates here....

Reply:
We can remove it as it is not consumed anywhere. I see only `modificationDate` getting consumed